package.json

```diff
@@ -188,6 +188,7 @@
 		"config": "^3.3.6",
 		"debug": "^4.3.3",
 		"enhanced-resolve": "5.9.3",
+		"fuse.js": "^6.6.2",
```

Comment:
I also think we should add it to `packages/search/package.json`

> You should add dependencies to the root project _only_ when it will be used to test and/or build other packages.

https://github.com/Automattic/wp-calypso/blob/trunk/docs/dependency-management.md

Reply:
We actually should add it to `packages/search` and remove it from here if you ask me.